src/test/java/lotto/domain/LottosTest.java

```diff
@@ -2,28 +2,43 @@
 
 import static org.junit.jupiter.api.Assertions.*;
 
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
-import org.assertj.core.api.Assertions;
-import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
 
 class LottosTest {
 
-    @DisplayName("당첨 번호가 4,5,6,7,8,9이고, 사용자가 구매한 로또가 1,2,3,4,5,6일 때 5등 당첨 확인")
     @Test
-    void 로또_당첨_확인() {
+    void 로또_생성_확인() {
         //given
-        Lotto winningLotto = new Lotto("4, 5, 6, 7, 8, 9");
-        String bonusNumber = "10";
-        WinningLotto prizeLotto = new WinningLotto(winningLotto, bonusNumber);
+        List<Integer> numbers = Arrays.asList(1,2,3,4,5,6);
+        FixedRandomNumber fixedRandomNumber = new FixedRandomNumber(numbers);
 
         //when
-        Lottos lottos = new Lottos(1, new FixedNumber());
+        Lottos lottos = new Lottos(5, fixedRandomNumber);
 
         //then
-        Assertions.assertThat(lottos.calculatePrize(prizeLotto).calculateProfit(new Money("10000")))
-                .isEqualTo(0.50);
+        assertEquals(5, lottos.getLottos().size());
+    }
+
+    @Test
+    void 당첨_결과() {
+        //given
+        List<Integer> numbers = Arrays.asList(1,2,3,4,5,6);
```

Comment:
사소한데 컨벤션을 지키면 좋을 것 같아요~

```suggestion
    @Test
    void 당첨_결과() {
        //given
        List<Integer> numbers = Arrays.asList(1, 2, 3, 4, 5, 6);
```

`Arrays#asList` 와 `List#of` 는 어떤 차이가 있을까요?

Reply:
헉,, 컨벤션을 의식하면서 작성을 하고 있는데 자꾸 놓치는게 있네요 ㅠㅠ 감사합니다.

제 생각에는 반환된 리스트의 불변성인 것 같아요!